services/worker-manager/src/worker-scanner.js

```diff
@@ -56,7 +56,7 @@ class WorkerScanner {
     const fetch =
       async (size, offset) => await this.db.fns.get_non_stopped_workers_quntil_providers(
         null, null, null, this.providersFilter.cond, this.providersFilter.value, size, offset);
-    for await (let row of paginatedIterator({ fetch })) {
+    for await (let row of paginatedIterator({ fetch, size: 500 })) {
```

Comment:
reducing size of batches a bit to also lower the risk of records getting outdated by the time they are checked